lib/insert/insert_refunds.rb

```diff
@@ -43,7 +43,7 @@ def self.with_stripe(charge, h)
 
     gross = -(h['amount'])
     platform_fee = BillingPlans.get_percentage_fee(charge['nonprofit_id'])
```

Comment:
I think we are not using this anymore, can I remove it?

Reply:
What aren't we using anymore?

Reply:
When you say we're not using it, is platform_fee not being used anywhere?

Reply:
Yes, it's not being used anywhere

Reply:
Ah, yes, you can take remove it.